CHANGES.rst

```diff
@@ -8,6 +8,106 @@ https://hyperspy.readthedocs.io/en/latest/user_guide/changes.html
 
 .. towncrier release notes start
 
+Hyperspy 1.7.0 (2022-04-26)
+===========================
+
+New features
+------------
+
+- Add ``filter_zero_loss_peak`` argument to the :py:meth:`~._signals.eels.EELSSpectrum.spikes_removal_tool` method (`#1412 <https://github.com/hyperspy/hyperspy/issues/1412>`_)
+- Add :py:meth:`~._signals.signal2d.Signal2D.calibrate` method to :py:class:`~._signals.signal2d.Signal2D` signal, which allows for interactive calibration (`#1791 <https://github.com/hyperspy/hyperspy/issues/1791>`_)
+- Add :py:meth:`~._signals.eels.EELSSpectrum.vacuum_mask` method to :py:class:`~._signals.eels.EELSSpectrum` signal (`#2183 <https://github.com/hyperspy/hyperspy/issues/2183>`_)
```

Comment:
the links to the method and class do not work

Reply:
For some reason only `LazyEELSSpectrum` is in the API-docs, not `EELSSpectrum`:
https://hyperspy--2932.org.readthedocs.build/en/2932/api/hyperspy._signals.eels.html

Reply:
Same as above.